javascript/chronometer.js

```diff
@@ -1,34 +1,57 @@
 class Chronometer {
   constructor() {
-    // ... your code goes here
+    this.currentTime = 0;
+    this.intervalId = null;
   }
 
   start(callback) {
-    // ... your code goes here
+    this.intervalId = setInterval (() => {
+      if (callback !== undefined) {
+        callback();
+      }
+      this.currentTime++
+    }, 1000)
   }
 
   getMinutes() {
-    // ... your code goes here
+    let minutes = 0;
+    if (this.currentTime >= 60) {
+      minutes++
+    }
+    return Math.round(this.currentTime / 60);
```

Comment:
Hi Dani, I think you saw that in class: just change Math.round to Math.floor:
`return Math.floor(this.currentTime / 60);`